src/views/studio/l10n.json

```diff
@@ -136,5 +153,7 @@
     "studio.alertCuratorInvited": "Curator invite sent to \"{name}\"",
     "studio.alertManagerPromote": "\"{name}\" is now a manager",
     "studio.alertManagerPromoteError": "Something went wrong promoting \"{name}\"",
-    "studio.alertMemberRemoveError": "Something went wrong removing \"{name}\""
+    "studio.alertMemberRemoveError": "Something went wrong removing \"{name}\"",
+    "studio.alertTransfer": "\"{name}\" is now the host",
+    "studio.alertTransferRateLimit": "You can only change the host once a day. Try again tomorrow."
```

Comment:
I think this message might be either somewhat misleading or difficult to track on the backend.

If it means there is a 24-hour cooldown, it might be misleading since it could already be "tomorrow" and you still couldn't change the studio host. For example, if you change host at 6 PM your time and want to change the host again at noon the next day.

If it means that it is actually just the next day, verifying time zones (and making sure it isn't just a limit bypass) might be difficult to track since the backend is only in one time zone.